openshift/productization/generate-dockerfiles/gen_dockerfiles.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -x
+
+for component in serving; do
+    for subcomponent in controller autoscaler activator networking-istio networking-certmanager webhook queue; do
```

Comment:
Can we possibly integrate this with the existing Makefile to generate all the Dockerfiles both for productization and CI, or even better: Can we use just one set of Dockerfiles for both?

Reply:
It's my current understanding that `go-toolset` is required to produce an "official" build. I cannot see any reference to `toolset` in https://github.com/openshift/ci-tools, or https://github.com/openshift/knative-serving. So at least for the for the time being, it looks like the productization and CI Dockerfiles will have to be separate. I'll update the Makefile with the appropriate changes.